robust_pdf_plot.ipynb

```diff
@@ -0,0 +1,311 @@
+{
```

Comment:
<p>Add meaningful title in 1st markdown cell, refer textbook for the same</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Done</p>